app/controllers/concerns/questionnaire_renderable.rb

```diff
@@ -5,7 +5,7 @@ def prep_questionnaire(json: true)
     @attrib = params[:filter] || "criteria"
     @response_set ||= @loan.send(@attrib) || ResponseSet.new(kind: @attrib, loan: @loan)
     @response_set.current_user = current_user
```

Comment:
do we still need current user on response_set?

Reply:
Gosh I don't think so! Great catch! I'll remove it.